projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-write/query-write-raise-query/query-write-raise-query.component.html

```diff
@@ -1,150 +1,108 @@
+          <div class="govuk-label-wrapper">
+            <label class="govuk-label govuk-label--m govuk-!-font-weight-bold" for="query-raise-subject">
+              {{ 'Query subject' | rpxTranslate }}
+            </label>
+          </div>
+          <div id="query-raise-subject-hint" class="govuk-hint">
+            {{ 'The subject should be a summary of your query' | rpxTranslate }}
           </div>
+          <p id="query-raise-subject-error" class="govuk-error-message"
+             *ngIf="formGroup.get('body')?.errors?.length">
+            <span class="govuk-visually-hidden">{{ 'Error:' | rpxTranslate }}</span> {{ 'Enter a subject' |
+            rpxTranslate }}
+          </p>
+          <input class="govuk-input" id="query-raise-subject" name="query-raise-subject" type="text"
+                 [class.govuk-input--error]="formGroup.get('subject')?.errors?.length"
+                 [formControlName]="'subject'"/>
+        </div>
 
-          <div class="govuk-form-group"
-               [class.govuk-form-group--error]="formGroup.get('body')?.errors?.length">
-            <div class="govuk-label-wrapper">
-              <label class="govuk-label govuk-label--m govuk-!-font-weight-bold" for="query-raise-body">
-                {{ 'Query body' | rpxTranslate }}
-              </label>
-            </div>
-            <div id="query-raise-body-hint" class="govuk-hint">
-              {{ 'Include as many details as possible so case workers can respond to your query' | rpxTranslate }}
-            </div>
-            <p id="query-raise-body-error" class="govuk-error-message"
-               *ngIf="formGroup.get('body')?.errors?.length">
-              <span class="govuk-visually-hidden">{{ 'Error:' | rpxTranslate }}</span> {{ 'Enter a body' |
-              rpxTranslate }}
-            </p>
-            <textarea [formControlName]="'body'" class="govuk-textarea"
-                      [class.govuk-textarea--error]="formGroup.get('body')?.errors?.length" id="query-raise-body"
-                      name="query-response" rows="5" aria-describedby="query-response-hint query-response-error">
-          </textarea>
+        <div class="govuk-form-group"
+             [class.govuk-form-group--error]="formGroup.get('body')?.errors?.length">
+          <div class="govuk-label-wrapper">
+            <label class="govuk-label govuk-label--m govuk-!-font-weight-bold" for="query-raise-body">
+              {{ 'Query body' | rpxTranslate }}
+            </label>
+          </div>
+          <div id="query-raise-body-hint" class="govuk-hint">
+            {{ 'Include as many details as possible so case workers can respond to your query' | rpxTranslate }}
           </div>
+          <p id="query-raise-body-error" class="govuk-error-message"
+             *ngIf="formGroup.get('body')?.errors?.length">
+            <span class="govuk-visually-hidden">{{ 'Error:' | rpxTranslate }}</span> {{ 'Enter a body' |
```

Comment:
@MateiHMCTS - I think this error message should probably read `Enter a query body` as opposed to `Enter a body`